src/components/TodoList.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+// import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
```

Comment:
when you have no logic in component and just return jsx you can delete the word `return ` and just return jsx in round brackets

`}) => (`

Reply:
Yes, you are right. I always write return because have some problem with syntax and this way help me.